### PR TITLE
Telemetry: Some changes in the telemetry reports we create

### PR DESCRIPTION
**Pull request by otaviobp**

### CODE REVIEW

**Comment by otaviobp** [comment]
@mbelluzzo: following your suggestions and some other improvements.
@jurobystricky and @alexjch: This may interest you too. I am changing the severity of some telemetry records created by swupd, so if you are looking at some swupd records you may be affected by this PR.

File: src/telemetry.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+extern "C" {
+#endif
+
+/**
+ * Severity of a telemetry record
+ */
+enum telemetry_severity {
+	/** @brief Temetry severity DEBUG */
+	TELEMETRY_DEBG = 1,
+	/** @brief Temetry severity DEBUG */
```
Comment (mbelluzzo) [inline src/telemetry.h:19]: Just missing to update the @brief lines for each entry.

**Comment by alexjch** [comment]
@otaviobp this is transparent to telemetry, the change is fine with me.

File: src/telemetry.c [hunk 2]
```diff
@@ -32,6 +32,16 @@
 
 #define RECORD_VERSION 2
 
+#ifdef DEBUG_MODE
```
Comment (castulo) [inline src/telemetry.c:35]: nice, good idea

**Pull request merged**